cypress-io/cypress
# feat(unify): share header bar between launchpad and app

- Closes https://cypress-io.atlassian.net/browse/UNIFY-500

### Additional details

A lot of this is file moves and adjustments to get the Header Bar to work correctly in the frontend-shared package. Part of this is allowing graphql codegen to run for Vue components inside frontend-shared, but only if they are nested in the /frontend-shared/src/gql-components directory. This is to prevent mixing gql-dependent components with shared components that can be shared beyond gql-driven packages like App and Launchpad.


### How has the user experience changed?


https://user-images.githubusercontent.com/8340719/139134617-64945f47-c808-45d8-90a1-091cc4dff0fa.mov



## How to Test

Use both the launchpad and the new runner and see that the header bar is present. See that in the App, a page name is shown and in the Launchpad, the "Projects > project-name" text appears. Change the browser being used via the dropdown in the header and see that the browser closes and the correct browser opens. Note that some things are still unfinished, like the login flow in app, which is currently not nailed down in terms of requirements, and the Cypress version list, with is being updated by other tickets.

### PR Tasks
